students/Ben_freeman/lesson05/test.py

```diff
@@ -0,0 +1,30 @@
+# pylint: disable=unused-import, unused-wildcard-import,
+# pylint: disable=too-few-public-methods, too-many-arguments, wildcard-import
+# pylint: disable=logging-format-interpolation, pointless-string-statement
+import sys
+import unittest
+import database as db
+from unittest.mock import patch
+
+
+class TestFunctions(unittest.TestCase):
+
+    def setUP(self):
+        with patch("builtins.input", side_effect="yes"):
+            db.import_data("data_files", "products", "customers", "rentals")
+
+    def test_import_data(self):
+        with patch("builtins.input", side_effect="yes"):
```

Comment:
Same as above, there will be two calls to input, so in this case, this line should be:

`with patch("builtins.input", side_effect=("yes", "yes")):`

Or you could use _Mock()_ and set _input_ to always return "yes", no matter the number of calls. Either way works.